src/main/java/com/datalogics/pdf/samples/forms/ExportFormDataToCsv.java

```diff
@@ -0,0 +1,179 @@
+     * @throws PDFIOException there was an error reading or writing a PDF file or temporary caches
+     * @throws PDFInvalidDocumentException a general problem with the PDF document, which may now be in an invalid state
+     * @throws PDFUnableToCompleteOperationException the operation was unable to be completed
+     * @throws URISyntaxException a string could not be parsed as a URI reference
+     */
+    public static void main(final String[] args) throws PDFInvalidDocumentException, PDFIOException,
+                    PDFSecurityException, IOException, PDFUnableToCompleteOperationException, URISyntaxException {
+        // If you are using an evaluation version of the product (License Managed, or LM), set the path to where PDFJT
+        // can find the license file.
+        //
+        // If you are not using an evaluation version of the product you can ignore or remove this code.
+        LicenseManager.setLicensePath(".");
+
+        URL inputUrl = null;
+        URL outputUrl = null;
+        if (args.length > 0) {
+            inputUrl = IoUtils.createUrlFromPath(args[0]);
+            outputUrl = IoUtils.createUrlFromPath(args[1]);
+        } else {
+            inputUrl = ExportFormDataToCsv.class.getResource(DEFAULT_INPUT);
+            outputUrl = IoUtils.createUrlFromPath(CSV_OUTPUT);
+        }
+
+        exportFormFields(inputUrl, outputUrl);
+    }
+
+    /**
+     * Export the form data to a comma separated form (CSV). The first row that is written out in the CSV contains the
+     * qualified field names and the second row contains the field values.
+     *
+     * @param inputUrl path to the input PDF file
+     * @param outputUrl path to write the CSV output file to
+     * @throws IOException an I/O operation failed or was interrupted
+     * @throws PDFSecurityException some general security issue occurred during the processing of the request
+     * @throws PDFIOException there was an error reading or writing a PDF file or temporary caches
+     * @throws PDFInvalidDocumentException a general problem with the PDF document, which may now be in an invalid state
+     * @throws PDFUnableToCompleteOperationException the operation was unable to be completed
+     * @throws URISyntaxException a string could not be parsed as a URI reference
+     */
+    public static void exportFormFields(final URL inputUrl, final URL outputUrl) throws PDFInvalidDocumentException,
+                    PDFIOException, PDFSecurityException, IOException, PDFUnableToCompleteOperationException,
+                    URISyntaxException {
+        final PDFDocument pdfDocument = DocumentUtils.openPdfDocument(inputUrl);
+
+        final PDFInteractiveForm form = pdfDocument.getInteractiveForm();
+
+        final File outputFile = new File(outputUrl.toURI());
+        if (outputFile.exists()) {
+            Files.delete(outputFile.toPath());
+        }
+
+        final PrintWriter writer = new PrintWriter(outputFile, "UTF-8");
```

Comment:
``PrintWriter`` implements ``AutoCloseable``, so use try-with-resources to close it``
(So does ``CSVPrinter``)